common/djangoapps/edxmako/paths.py

```diff
@@ -46,6 +47,18 @@ def add_directory(self, directory, prepend=False):
         self._collection.clear()
         self._uri_cache.clear()
 
+    def get_template(self, uri):
+        """
+        Overridden method which will allow us to look up templates from a database (optionally)
```

Comment:
We do not know where the template is coming from in this context, so we should instead say something along the lines of "Hand-off the template lookup to the microsite subsystem"